src/dex/maverick-v2/config.ts

```diff
@@ -0,0 +1,27 @@
+import { DexParams } from './types';
+import { DexConfigMap } from '../../types';
+import { Network } from '../../constants';
+
+export const MAV_V2_BASE_GAS_COST = 50_000;
+export const MAV_V2_TICK_GAS_COST = 10_000;
+
+export const MaverickV2Config: DexConfigMap<DexParams> = {
+  MaverickV2: {
+    [Network.BASE]: {
+      poolLensAddress: `0x56eFfDD51b20705e152CAF482D9A6972e97B571C`,
+      apiURL: 'https://maverick-v2-api-delta.vercel.app',
+    },
+    [Network.MAINNET]: {
+      poolLensAddress: `0x56eFfDD51b20705e152CAF482D9A6972e97B571C`,
+      apiURL: 'https://maverick-v2-api-delta.vercel.app',
+    },
+    [Network.BSC]: {
+      poolLensAddress: `0x56eFfDD51b20705e152CAF482D9A6972e97B571C`,
+      apiURL: 'https://maverick-v2-api-delta.vercel.app',
+    },
+    [Network.ARBITRUM]: {
+      poolLensAddress: `0x56eFfDD51b20705e152CAF482D9A6972e97B571C`,
+      apiURL: 'https://maverick-v2-api-delta.vercel.app',
```

Comment:
addresses are updated at https://github.com/maverickprotocol/maverick-v2-examples/tree/main?tab=readme-ov-file#mainnets

lens is now 0x942646b0A8B42Af1e1044439013436a9a3e080b5 for all these chains.

Reply:
also, url is https://v2-api.mav.xyz/

Reply:
What about Router addresses?

Reply:
Due to technical difficulties, we decided to use the MaverickRouter for this integration

Reply:
the router contracts are correct at https://github.com/maverickprotocol/maverick-v2-examples/tree/main?tab=readme-ov-file#mainnets

Feel free to use those